### ISSUE TITLE
gazebo: give 'remap' launch arg a more descriptive name

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/staubli_experimental#7]
The `*_gazebo.launch` files currently expose a `remap` argument that, when `true`, remap the default topics from the `gazebo_ros` plugin (and friends) to the ROS-Industrial driver spec compliant ones.

`remap` is too generic a name, and should be changed into something which more clearly conveys its purpose / effect.

**Comment by gavanderhoorn** [comment]
Launch files in `rx160_gazebo` and the to-be-introduced `tx90_gazebo` packages (see #6) are affected.

**Comment by nilsmelchert** [comment]
Good idea. Besides renaming "remap" to something more meaningful, I think using the ROS-Industrial driver spec compliant topics by default would be a good idea. Since this is a ROS-Industrial repo, it makes a lot of sense to work with the industrial convention in gazebo by default.

**Comment by nilsmelchert** [comment]
What about extending the argument "remap" to something more meaningful like "ros_remapping", "ros_remap", "ros_i_remapping" or "ros_i_remap" and set the default value to true.

**Comment by nilsmelchert** [comment]
So if I did a PR for this, would you consider merging it? Or is there a reason not to do these changes?
